basket/news/forms.py

```diff
@@ -54,8 +54,7 @@ def to_python(self, value):
 
 def country_choices():
     """Upper and Lower case country codes"""
-    regions = product_details.get_regions('en-US')
-    return regions.items() + [(code.upper(), name) for code, name in regions.iteritems()]
+    return SFDC_COUNTRIES.items() + [(code.upper(), name) for code, name in SFDC_COUNTRIES.iteritems()]
```

Comment:
Just curious - why does this list need to contain both upper and lower case country codes?

Reply:
This was done a while ago, but my recollection is that it was the easiest way to have it accept either uppercase or lowercase country codes.